tip-wlan/charts/opensync-gw-cloud/values.yaml

```diff
@@ -13,6 +13,21 @@ image:
   name: opensync-gateway-cloud
   tag: 0.0.1-SNAPSHOT
 
+waitForServicesImage:
+  registry: docker.io
+  repository: opsfleet/depends-on
+  tag: latest
```

Comment:
@brennerm maybe we should consider pinning specific versions of these images? Things can unexpectedly break if we won't do that, especially in the alpine case below.

Reply:
Fixed